<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QMainWindow" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="DatosVehiculo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar Datos:</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>681</width>
      <height>401</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>679</width>
       <height>399</height>
      </rect>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>681</width>
        <height>401</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Placa</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Marca</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Modelo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cilindraje</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo de uso</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Combustible</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. Pasajeros</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. Chasis</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>No. Motor</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>251</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
